More Vending Machines take2

BloodyMan opened the pull request:
:cl: Cajoes
rscadd: Snix, HotFoods and Lavatory vending machines and hid them on the map.
tweak: All snack food relative paths are now absolute paths. Nobody noticed nor cared.
/:cl:

babydoll commented:
Maps are certainly not correct.

BloodyMan commented:
Well I don't know what to tell you. I did all the steps outlined in the [MapMerge Guide](https://baystation12.net/forums/threads/setting-up-using-maptools-ii-the-curious-case-of-mapmerge2.6314/) to the letter.

BloodyMan commented:
Had to outsource it, but the maps are clean now. No idea what was hiccuping here.

babydoll commented:
All the `New()` that you converted in snacks.dm have double indents

BloodyMan commented:
And you could not mentioned this before because...?

babydoll commented:
I didn't look before.

babydoll commented:
FYI and FWIW if you're adding something, don't go and make a 700~ line change to a file just because you can. It's appreciated if you want to re-format a file into our preferred method, but it also makes it a lot bigger hassle to review.

ChaosAlpha committed:
adds 3 vending machines

No memory of changing this, reverting.

I do not remember changing this

Typo

no memory altering this

Did not touch these afaik

where do these keep coming from

more bizarrenesss

Update .all_changelog.yml

Update changelog.html

Update changelog.html

Update changelog.html

thing is breaking down

Commit failed - exit code 1 received, with output: 'error: cannot spawn .git/hooks/pre-commit: No such file or directory'

Map Conversion to TGM

adds 3 vending machines

No memory of changing this, reverting.

I do not remember changing this

Typo

no memory altering this

Did not touch these afaik

where do these keep coming from

more bizarrenesss

Update .all_changelog.yml

Update changelog.html

Update changelog.html

Update changelog.html

thing is breaking down

Commit failed - exit code 1 received, with output: 'error: cannot spawn .git/hooks/pre-commit: No such file or directory'

maps

screw it

Revert "screw it"

This reverts commit b96df1e50604b9af3ff2bbd9c10b6ad414170969.

new maps

adds 3 vending machines

No memory of changing this, reverting.

I do not remember changing this

Typo

no memory altering this

Did not touch these afaik

where do these keep coming from

more bizarrenesss

Update .all_changelog.yml

Update changelog.html

Update changelog.html

Update changelog.html

thing is breaking down

Commit failed - exit code 1 received, with output: 'error: cannot spawn .git/hooks/pre-commit: No such file or directory'

Map Conversion to TGM

adds 3 vending machines

No memory of changing this, reverting.

I do not remember changing this

Typo

no memory altering this

Did not touch these afaik

where do these keep coming from

more bizarrenesss

Update .all_changelog.yml

Update changelog.html

Update changelog.html

Update changelog.html

thing is breaking down

Commit failed - exit code 1 received, with output: 'error: cannot spawn .git/hooks/pre-commit: No such file or directory'

maps

screw it

Revert "screw it"

This reverts commit b96df1e50604b9af3ff2bbd9c10b6ad414170969.

new maps

boop

Change to species blurb generation

Moved it into species datum, added more information to it.
Adds random alien humanoid species

Just a greyscaled human sprite that can be colored.
Their properties are randomized when species is created.
Adds a language for them, low effort mash of bogani/cult/vox syllabi
Adds translator implant

Lets you learn languages by hearing them being spoken, currently need to hear it 20 times to learn it.
Updates preview icon

Added a proc to adapt species to desired atmo

Also verb to show info about own species

Removed rando closing center tag
Made species change apply default species color unless asked otherwise
Travis appeasing

Adds stuff

snacks indent refactor

indent fix

boop


Change to species blurb generation

Moved it into species datum, added more information to it.
Adds random alien humanoid species

Just a greyscaled human sprite that can be colored.
Their properties are randomized when species is created.
Adds a language for them, low effort mash of bogani/cult/vox syllabi
Adds translator implant

Lets you learn languages by hearing them being spoken, currently need to hear it 20 times to learn it.
Updates preview icon


Added a proc to adapt species to desired atmo

Also verb to show info about own species 

Removed rando closing center tag
Made species change apply default species color unless asked otherwise
Travis appeasing


Adds stuff

BloodyMan commented:
Indent situation rectified and maps have been remerged.
